docs/framework/unmanaged-api/profiling/icorprofilerinfo8-isfunctiondynamic-method.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "ICorProfilerInfo8::IsFunctionDynamic"
+ms.date: "08/06/2019"
+dev_langs: 
+  - "cpp"
+api_name: 
+  - "ICorProfilerInfo8.IsFunctionDynamic"
+api_location: 
+  - "mscorwks.dll"
+api_type: 
+  - "COM"
+author: "davmason"
+ms.author: "davmason"
+---
+# ICorProfilerInfo8::IsFunctionDynamic Method
+  
+  Determines if a function has associated metadata.    
```

Comment:
Might this not be better as "Determines if a function does not have associated metadata.", since that's the true value?

Reply:
Yes, that makes more sense